### ISSUE TITLE
Add Date facet and range form to results view

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#1435]
- add date facet as second facet (below Options) in the facets column
- add a date range form to the top of the facet panel
- see SW catalog date range fields for markup (including sr-only labels)
- populate the "from" field with the earliest year represented in the results, and the "to" field with the latest year
- user can change one or both values and click Apply to see filtered results

![screen shot 2017-06-30 at 11 25 00 am](https://user-images.githubusercontent.com/6945691/27749933-88ed8042-5d8a-11e7-983c-f4278e487d8c.png)

With date range applied:
- the from and to fields show the dates entered by the user 
- show the individual facet values represented in the results
- breadcrumb shows the range applied (i.e. not individual bubbles for each year in the range)

![screen shot 2017-06-30 at 11 55 40 am](https://user-images.githubusercontent.com/6945691/27750203-a46f9c5a-5d8b-11e7-80ff-c417c1b766c1.png)

![screen shot 2017-06-30 at 11 58 49 am](https://user-images.githubusercontent.com/6945691/27750207-a86e78da-5d8b-11e7-90e7-97aa4d5ae78c.png)

**Comment by jkeck** [comment]
I believe the ability to do this type of query will require some feature addition to the `ebsco-eds` gem.   We may want to bring this up w/ them on slack/GH to see where they might be able to fit this in (unless they already have support, which wasn't immediately obvious).

I worry that this might not be `Ready` based on the support for this feature in the gem.

**Comment by jvine** [comment]
@jkeck can we split the default facet (list of years) into a separate ticket?

**Comment by drh-stanford** [comment]
PR #1520 provides `eds_publication_year_range_facet` so I think this is unblocked if that PR gets merged.

**Comment by jkeck** [comment]
This is currently blocked by needing a min/max range from the EDS response.  The current limitation of 50 items returned as a facet will require something custom to generate a proper slider UI.

**Comment by jvine** [comment]
Closing, unblocked by Ebsco, design replaced by #1609
